api/IDBObjectStore.json

```diff
@@ -966,10 +966,10 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "7"
+              "version_added": "10.1"
```

Comment:
This is `openKeyCursor` and is later than the rest. Is this based on mdn-bcd-collector results?

Reply:
Yeah, this is pretty much all from the collector results, aside from ensuring iOS stays marked as "8" and not earlier.

Reply:
This doesn't match https://developer.mozilla.org/en-US/docs/Web/API/IDBIndex/openKeyCursor, can you check if the data for that is also wrong?

Reply:
OK, I've compared https://mdn-bcd-collector.appspot.com/tests/api/IDBObjectStore and https://mdn-bcd-collector.appspot.com/tests/api/IDBIndex in Safari 10 and 10.1, and there's a lot more than this that came in 10.1.

How about backing this out and making all of those changes as a single PR? I'm not sure I understand why the collector isn't updating all of the data, but it looks like api.IDBIndex.openKeyCursor was in Safari 8 and just temporarily gone. Something weird happened at any rate, which is better to split out.